### PR TITLE
chore: Website add product updates #7

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: website/pages/blog/product-updates-7.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- Open source: SDKs & Plugins.
+- Premium (closed source): New Plugins, Policies and dashboards.
+
+## SDKs!
+
+With recent Adoption of [Apache Arrow](https://www.cloudquery.io/blog/adopting-apache-arrow-at-cloudquery) as CloudQuery
+in memory data form CloudQuery SDK is now expanding beyond Go (which means you can write CloudQuery plugins in other languages)! The first one is of-course the king of data engineering - Python!
+
+Check out our [Python announcement blog](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk)!
```
Comment (erezrokah) [inline website/pages/blog/product-updates-7.mdx:26]: We could link to https://www.cloudquery.io/docs/developers/creating-new-plugin/python-source and https://www.cloudquery.io/docs/developers/creating-new-plugin/javascript-source too

File: website/pages/blog/product-updates-7.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* [Jira](https://www.cloudquery.io/integrations/jira)
+* [MongoDB Atlas](https://www.cloudquery.io/integrations/mongodb-atlas)
+* [Render](https://www.cloudquery.io/integrations/render)
+
+### AWS, GCP, Azure
+
+Checkout full changelog for [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md),
+[GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md)
+for full updates.
```
Comment (erezrokah) [inline website/pages/blog/product-updates-7.mdx:47]:
```suggestion
Checkout full changelog for [AWS](https://github.com/cloudquery/cloudquery/blob/97d86a5c282e587420c1abd714bb0218d9c86f5a/plugins/source/aws/CHANGELOG.md),
[GCP](https://github.com/cloudquery/cloudquery/blob/97d86a5c282e587420c1abd714bb0218d9c86f5a/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/97d86a5c282e587420c1abd714bb0218d9c86f5a/plugins/source/azure/CHANGELOG.md)
for full updates.
```

We should pin the links to the commit hash

**Review by erezrokah — COMMENTED** [review]
Added a few comments. Looks great 🚀

File: website/pages/blog/product-updates-7.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+TLDR:
+
+- Open source: SDKs & plugins.
+- Premium (closed source): New plugins, policies and dashboards.
+
+## SDKs!
+
+With recent Adoption of [Apache Arrow](https://www.cloudquery.io/blog/adopting-apache-arrow-at-cloudquery) as CloudQuery
+in memory data form CloudQuery SDK is now expanding beyond Go (which means you can write CloudQuery plugins in other languages)! The first one is of-course the king of data engineering - Python!
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-7.mdx:24]:
```suggestion
in-memory data format, the CloudQuery SDK is now expanding beyond Go (which means you can write CloudQuery plugins in other languages)! The first one is, of course, the king of data engineering--Python!
```

File: website/pages/blog/product-updates-7.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Check out our [Python announcement blog](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk)!
+
+## Sources
+
+### New Open Source Sources
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-7.mdx:30]:
```suggestion
### New Open Source Sources Plugins
```

**Review by hermanschaaf — APPROVED** [review]
Great stuff! 🚀

File: website/pages/blog/product-updates-7.mdx (hunk 1 above)
Comment (yevgenypats) [reply website/pages/blog/product-updates-7.mdx]: Nice! I wasn't aware javascript already had a guide - we should also schedule some socials around that.

File: website/pages/blog/product-updates-7.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+## SDKs!
+
+With the recent [Adoption of Apache Arrow](https://www.cloudquery.io/blog/adopting-apache-arrow-at-cloudquery) as CloudQuery's
+in-memory data format, the CloudQuery SDK is now expanding beyond Go (which means you can write CloudQuery plugins in other languages)!
+
+The first one is, of course, the undisputed king of data engineering--Python! and the 2nd after is the undisputed king of the web--JavaScript!
+
+* Check out our [Python announcement blog](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk)!
+* Creating a [Python source](https://www.cloudquery.io/docs/developers/creating-new-plugin/python-source)
+* Creating a [Javascript source](https://www.cloudquery.io/docs/developers/creating-new-plugin/javascript-source)
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-7.mdx:30]:
```suggestion
* Creating a [JavaScript source](https://www.cloudquery.io/docs/developers/creating-new-plugin/javascript-source)
```

**Pull request merged**